planningcenter/topbar/modules/apps_provider.tsx

```diff
@@ -1,11 +1,12 @@
 import * as React from "react";
 import pcoUrl from "./pco_url";
-import getJSON from "./get_json";
+import apiRequest, { Fetch, defaultFetch } from "./api_request";
 
 export class AppsProvider extends React.Component<
   {
     env: string;
     formatter?: any;
+    authenticatedFetch?: Fetch;
```

Comment:
Authentication is the current goal of the consumer defining their own `fetch`. I wonder if that will always be the case? I find myself wanting to call this `configuredFetch`, or maybe even `overridenFetch`, to open up the possibilities of doing _whatever_ you want to do by intercepting the `fetch` arguments before it's in flight.

Reply:
`configuredFetch` seems like a great option here. It leaves that door open to anything else the host app might need to do to make a good request. Thanks for that suggestion

Reply:
i too prefer `configuredFetch` to `overridenFetch`.
i think the connotation is better — more closely aligned to the actual pattern: injected fetch

Reply:
`configuredFetch` is in the latest change